### DISCUSSION TITLE
How to play VP9 Opus?

### DISCUSSION THREAD

**Discussion by gerroon** [jellyfin/jellyfin#10541] [Q&A]
Hi

I download videos from Youtube and they seem to be VP9 Opus. I am using the official Docker on Debian and IOS 17.1 I installed the Swiftfin and the Jellyfin mobile. Neither of them can play these videos. Any recommendations to play these?

**Comment by pedrojreis** [comment]
Hey @gerroon 

[this](https://jellyfin.org/docs/general/clients/codec-support/) page might be useful for you.

![image](https://github.com/jellyfin/jellyfin/assets/28795057/d2a60564-0c6c-4b90-9235-dc517c31b995)

It seems iOS does not really like VP9 :/ You can always try and convert your videos, for example with [handbrake](https://handbrake.fr/)
